### ISSUE TITLE
Docker. Styles not loaded.

### ISSUE THREAD

**Issue by j0k3r** [wallabag/docker#115]
_From @kv-gits on June 19, 2018 7:23_

### Issue details

Run by the docs:
```
docker run -v /opt/wallabag/data:/var/www/wallabag/data -v /opt/wallabag/images:/var/www/wallabag/web/assets/images -p 80:80 wallabag/wallabag
```

Styles do not work

### Environment
wallabag version - latest from "docker pull"

### Steps to reproduce/test case

run docker container, go to localhost

_Copied from original issue: wallabag/wallabag#3676_

**Comment by j0k3r** [comment]
This does not enable or build assets.
It just define the folder where image will be stored for each article.

What error do you have about your _Styles_?

**Comment by kv-gits** [comment]
I see no styles - only raw html on service page. No css applied. So usage is practically unavailable.

**Comment by j0k3r** [comment]
I can understand thanks.
What error do you have? Is it a 404 on each css request?
Did you follow all steps about using docker and installing wallabag?

**Comment by kv-gits** [comment]
It tryes to get styles (etc) from https://your-wallabag-url-instance.com/wallassets/material.css

Seems I missed some steps in config. I tried by the docs: "simpliest method to run with sqlite"

**Comment by kv-gits** [comment]
Tried this way
```
sudo docker run -v /home/a1/dockerobjects/wallabag/data:/var/www/wallabag/data -v /home/a1/dockerobjects/wallabag/images:/var/www/wallabag/web/assets/images -e SYMFONY__ENV__DOMAIN_NAME="https://127.0.0.1:8090"  -p 8090:80 wallabag/wallabag
```
Style also not loaded request status in dev-tools is empty

**Comment by kv-gits** [comment]
Yes, just need to add one env variable ```-e SYMFONY__ENV__DOMAIN_NAME="http://127.0.0.1:8090"```

**Comment by j0k3r** [comment]
Fixed?

**Comment by kv-gits** [comment]
yes

**Comment by FPMilan** [comment]
@kv-gits how exactly did you fix it?
I just appended 
SYMFONY__ENV__DOMAIN_NAME="http://127.0.0.1:8090" 
to /etc/environment but it did´nt change anything. Did I do it wrong?

**Comment by fviccia** [comment]
Thanks for this issue - it helped me solve the exact same problem! I was experiencing the ugly HTML-only view when running Wallabag 2.6.14 in Docker, even though the container was running
correctly.

The Fix: Remove the images bind mount from your docker-compose.yml:

# BEFORE (broken - causes CSS not to load)
volumes:
  - /path/to/wallabag/data:/var/www/wallabag/data
  - /path/to/wallabag/images:/var/www/wallabag/web/assets/images  # ❌ Remove this

# AFTER (fixed)
volumes:
  - /path/to/wallabag/data:/var/www/wallabag/data  # ✅ Keep only data bind mount

Why This Works: The images bind mount to /var/www/wallabag/web/assets/images/ conflicts with Wallabag's asset serving mechanism. By removing it, Wallabag can properly serve CSS/JS files fromits internal filesystem while still maintaining the critical database bind mount.

Trade-off:

  • ✅ CSS/JS loads correctly - Wallabag looks proper
  • ✅ Database still safely bind-mounted and backed up
  • ⚠️  Uploaded images stored in container filesystem (not critical for most use cases)

This fix worked for Wallabag 2.6.14 on Docker Compose. Hope this helps others facing the same issue!
